src/dns/aardvark.rs

```diff
@@ -313,7 +315,7 @@ impl Aardvark {
         let mut modified = false;
         let container_id = network_options.container_id;
         for (key, network) in network_options.network_info {
-            if network.dns_enabled {
+            if network.dns_enabled && network.driver != DRIVER_MACVLAN {
```

Comment:
I thought about this and lets make it == bridge instead, if we add more drivers this will bite us again, better to explicitly opt in.